src/waveform/renderers/glslwaveformrenderersignal.cpp

```diff
@@ -297,6 +297,7 @@ void GLSLWaveformRendererSignal::draw(QPainter* painter, QPaintEvent* /*event*/)
             m_waveformRenderer->getFirstDisplayedPosition() * dataSize / 2.0);
     const auto lastVisualIndex = static_cast<GLfloat>(
             m_waveformRenderer->getLastDisplayedPosition() * dataSize / 2.0);
+    qWarning() << firstVisualIndex << lastVisualIndex;
```

Comment:
accidentally committed debugging info?

Reply:
Yes. But it doesn't make sense to review this PR on such a level right now. The whole C++ part to get the waveform texture into QML is a giant hack and basically unmergeable anyway.

Reply:
Yeah, it was just something that stood out from a quick glance at the code